### ISSUE TITLE
Streaming API: Explore upper limits

### ISSUE THREAD

**Issue by dschoeni** [gmazlami/adv_se_project#6]
Find out whether putting in more phrases affects the output of the streaming API (is it a bottleneck or not?).

**Comment by LilianaB** [comment]
More phrases == more tweets. The bottleneck if any will be in the client (processing the tweets).  I did a test with the worldwide trending topics today. I used the following query:
stream = tweeter.stream('statuses/filter', {
 'track': "SkrillexNoMultiShow, ULTRALIVE, UTNWillyToledo, Vote1DUК, PraNamorarComigoPrecisa, Dusty Rhodes, KOR Ver, Ask Nash & Skylynn, Zayn and Naughty Boy, Riaño",
 language : 'en',
 stall_warnings: true
}
I got 9000 tweets in 5 minutes. Turns out that if your client is too slow in processing the tweets you will be disconnected. It is possible to get a warning message before they disconnect you if stall_warning is set to true. For more info about how does the stall_warning works check https://dev.twitter.com/streaming/overview/request-parameters#stallwarnings
 
In my test I did not get any warning, basically my clients does nothing, only counts tweets. We have to check if the sentimental analysis causes bottleneck.

Something to consider is that we can only track 400 keywords at the time, so there is some limitation there. The phrase's size is also limited, allowed size is [1-60] bytes
